netutils/config/parser.py

```diff
@@ -375,7 +376,7 @@ def build_config_relationship(self):
 
         return self.config_lines
 
-    def _build_multiline_config(self, delimiter):
+    def _build_multiline_config(self, delimiter: str):  # type: ignore
```

Comment:
Resolvable through:
- Adding `-> t.Optional[ConfigLine]` to the function signature
- Add an explicit default `return None` at the end
- Checking for None in all the places this function is called from